### DISCUSSION TITLE
10.000 pulse

### DISCUSSION THREAD

**Discussion by BFipsen** [klaasnicolaas/home-assistant-glow#150] [Q&A]
I have home assistant glow up and running, and i have put the home-assistant-glow.yaml file in the config/esphome folder and changed the walue from 1000 to 10000, but it looks like the plugin dont read the file, am i missing some thing?

**Comment by klaasnicolaas** [comment]
Has your problem already been solved? If not, could you describe more clearly where exactly it goes wrong? If you adjust the pulse value at the top of the YAML file and then flash the new firmware on your glow it should be resolved 🤔
